arduino/cores/packagemanager/loader.go

```diff
@@ -319,6 +320,10 @@ func (pm *PackageManager) loadPlatformRelease(platform *cores.PlatformRelease, p
 		return fmt.Errorf("loading %s: %s", platformTxtLocalPath, err)
 	}
 
+	if platform.Properties.SubTree("discovery").Size() > 0 {
+		platform.PluggableDiscoveryAware = true
+	}
```

Comment:
```suggestion
	platform.PluggableDiscoveryAware = platform.Properties.SubTree("discovery").Size() > 0
```

Reply:
nah, it's more readable as it is, I won't change it